File: LLM-Translations/GPT-4/codenet/C/Python/s141680236.py

def push(stack, chonda):
    stack.append(chonda)

def pop(stack):
    return stack.pop()

def main():
    stack = []
    while True:
        try:
            null = input()
        except EOFError:
            break

        if null == '+':
            chonda = pop(stack)
            yamagami = pop(stack)
            push(stack, chonda + yamagami)
        elif null == '-':
            yamagami = pop(stack)
            chonda = pop(stack)
            push(stack, chonda - yamagami)
        elif null == '*':
            chonda = pop(stack)
            yamagami = pop(stack)
            push(stack, chonda * yamagami)
        else:
            push(stack, int(null))

    print(pop(stack))

if __name__ == "__main__":
    main()

File: LLM-Translations/GPT-4/codenet/C/Java/s141680236.java

import java.util.Scanner;
import java.util.Stack;

public class s141680236 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        Stack<Integer> stack = new Stack<>();

        while (scanner.hasNext()) {
            String input = scanner.next();

            if (input.equals("+")) {
                int yamagami = stack.pop();
                int chonda = stack.pop();
                stack.push(chonda + yamagami);
            } else if (input.equals("-")) {
                int yamagami = stack.pop();
                int chonda = stack.pop();
                stack.push(chonda - yamagami);
            } else if (input.equals("*")) {
                int yamagami = stack.pop();
                int chonda = stack.pop();
                stack.push(chonda * yamagami);
            } else {
                stack.push(Integer.parseInt(input));
            }
        }

        System.out.println(stack.pop());
    }
}

// 
